setup.py

```diff
@@ -33,17 +32,13 @@ def run(self):
         'License :: OSI Approved :: BSD License',
         'Operating System :: OS Independent',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 3.4',
-        'Programming Language :: Python :: 3.5',
```

Comment:
supporting Python 3.5 is more effort than benefit; decreasing users, Django has dropped support in new versions

Reply:
Agreed. Let's add a [python_requires](https://packaging.python.org/guides/distributing-packages-using-setuptools/#python-requires) parameter so pip will install the right version.

Reply:
This does not deprecate support - AFAIK, this removes support, as pip will not find the release anymore on PyPi.

Reply:
Yeah, I think removing was the intention given that 3.4 has already reached EOL and 3.5 is impracticable to support.